torch/_prims/nvfuser_executor.py

```diff
@@ -26,6 +26,13 @@
 else:
     DataType = None
 
+import os
+
+dump_nvtx = False
+if os.getenv("PYTORCH_NVFUSER_DUMP_NVTX"):
```

Comment:
Let's do this in a function instead. This would allow us the set the env variable inside a user script.

e.g.
```
@lru_cache(None)
def get_nvprim_dump_nvtx():
  return os.getenv("PYTORCH_NVFUSER_DUMP_NVTX")
```

Reply:
smart, will change this